cli/register/servicewait.go

```diff
@@ -148,32 +307,44 @@ func WaitForService(org string, waitService string, waitTimeout int, pattern str
 		}
 		msgPrinter.Println()
 	} else {
-		msgPrinter.Printf("The following errors were found in the node's event log and are related to %v/%v. Use 'hzn eventlog list -s severity=error -l' to see the full detail of the errors.", org, waitService)
-		msgPrinter.Println()
+		for _, ss := range servSpecArr {
```

Comment:
I think we need to check ss.Status before looking for event log messages. If the service started (i.e. the status is good) then we dont need to check the event log for errors related to that service.

Reply:
Not sure what you mean by the service is good, but I made changes based on what I think you're trying to say. Let me know if it's not quite what you want.

Reply:
Perhaps another misunderstanding on my part. ss.Status seems to indicate that the service was started or not (ss.status == -1). We only want to extract log records from the error log and display for the cases where the service fails to get into execution. Perhaps ss.status is the wrong way to discriminate these cases?

Reply:
Done. Added a check for ss.status != 1.